tox.ini

```diff
@@ -44,6 +44,7 @@ extras =
     vagrant
     windows
 commands_pre =
+    ansible28: pip install --pre ansible==2.8b1
```

Comment:
Replace it with `ansible>=2.8b1,<2.9` and it will be ok, as we do not want to bump it on each new pre-release.

Reply:
I also hope that final merge does not contain emoji. I don't mind them on Github, but I do not want to see them in commit messages.

Reply:
FTR https://github.blog/2018-10-16-future-of-software/#suggested-changes-public-beta